pytest/lib/branches.py

```diff
@@ -87,6 +87,8 @@ def download_binary(uname, branch):
     """Download binary for given platform and branch."""
     url = f'https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore/{uname}/{branch}/near'
     print(f'Downloading near & state-viewer for {branch}@{uname}')
+    # TODO: make platform-independent.
```

Comment:
you can check what the underlying OS is and do things accordingly, although there may be a better solution. @chefsale @frol

Reply:
Seems like '\=https' worked.

Reply:
Done